test/extended/testdata/builds/volumes/s2i-buildconfig.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+kind: List
```

Comment:
non-blocking nit - ditto on using a List that has only one item.

Reply:
Leftover from the yaml I copied it from that did not get cleaned up.

Reply:
updated in next push